src/views/v2/rsvp/details/availability.php

```diff
@@ -17,18 +17,53 @@
  * @version TBD
  */
 
-$is_unlimited = -1 === $rsvp->remaining();
+use Tribe__Date_Utils as Dates;
 
+$is_unlimited = - 1 === $rsvp->remaining();
+$is_in_stock  = $rsvp->is_in_stock();
+
+$days_to_rsvp = Dates::date_diff( current_time( 'mysql' ), $rsvp->end_date );
+$days_to_rsvp = floor( $days_to_rsvp );
+
+// Only show Days to RSVP if it is happening within the next week and is in stock.
+if ( ! $is_in_stock || 6 < $days_to_rsvp ) {
+	$days_to_rsvp = false;
+}
 ?>
 <div class="tribe-tickets__rsvp-availability tribe-common-h6 tribe-common-h--alt tribe-common-b3--min-medium">
-	<?php if ( ! $rsvp->is_in_stock() ) : ?>
+	<?php if ( ! $is_in_stock ) : ?>
 		<?php $this->template( 'v2/rsvp/details/availability/full', [ 'rsvp' => $rsvp ] ); ?>
 	<?php elseif ( $is_unlimited ) : ?>
-		<?php $this->template( 'v2/rsvp/details/availability/unlimited', [ 'is_unlimited' => $is_unlimited ] ); ?>
+		<?php
+		$this->template(
+			'v2/rsvp/details/availability/unlimited',
+			[
+				'is_unlimited' => $is_unlimited,
+				'days_to_rsvp' => $days_to_rsvp,
+			]
+		);
+		?>
 	<?php else : ?>
-		<?php $this->template( 'v2/rsvp/details/availability/remaining', [ 'rsvp' => $rsvp ] ); ?>
+		<?php
+		$this->template(
+			'v2/rsvp/details/availability/remaining',
+			[
+				'rsvp'         => $rsvp,
+				'days_to_rsvp' => $days_to_rsvp,
+			]
+		);
+		?>
 	<?php endif; ?>
 
-	<?php $this->template( 'v2/rsvp/details/availability/days-to-rsvp', [ 'rsvp' => $rsvp ] ); ?>
-
+	<?php if ( false !== $days_to_rsvp ) : ?>
+		<?php
+		$this->template(
+			'v2/rsvp/details/availability/days-to-rsvp',
+			[
+				'rsvp'         => $rsvp,
+				'days_to_rsvp' => $days_to_rsvp,
```

Comment:
`$days_to_rsvp` would be only needed in this template. I'll move the logic to that template, bail early if false, and/or include the `|` at the top (when it's the case). That way we can move logic out from this template (`availability.php`) and be consisten with the way we're building templates.

Reply:
Design QA decided to use comma, I moved the logic out of the template so it can be provided (so it can check for false) in the other templates that need to have the comma. Can't think of another way around it unfortunately.